<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EepromWB</class>
 <widget class="QWidget" name="EepromWB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(114, 159, 207);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 28pt &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>eeprom WB</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>361</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>endianness</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_wb_endianness"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>rValue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_rValue"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>gbValue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_gbValue"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>rOverGValue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_rOverGValue"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>grOverGBValue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_grOverGBValue"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>datatype</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_wb_datatype"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>grValue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_grValue"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>bValue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_bValue"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>bOverGValue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_bOverGValue"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>qValue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_qValue"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
